hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -47,6 +51,10 @@ public static HawkBuilder init(Context context) {
       return remove(key);
     }
 
+    if (!isHawkInitialised()) {
+      return false;
```

Comment:
Not sure what is the best thing to do here. would it be better if it throws an exception every time the user tries to use Hawk when is not initialise?